packages/react-combobox/src/components/Listbox/useListboxStyles.ts

```diff
@@ -7,11 +7,12 @@ export const listboxClassName = 'fui-Listbox';
  * Styles for the root slot
  */
 const useStyles = makeStyles({
+  // TODO: add themed styles
   root: {
-    // TODO Add default styles for the root element
+    boxShadow: '0px 0px 2px 0px #0000001F, 0px 8px 16px 0px #00000024',
+    ...shorthands.borderRadius('4px'),
+    backgroundColor: '#fff',
```

Comment:
`#fff` is definitely not theme aware, and you should use a design token for this